components/expansion-panel/src/vwc-expansion-panel.scss

```diff
@@ -64,11 +64,26 @@
 			margin-inline-start: 16px;
```

Comment:
any style related to the ghost appearance should be selected by
```css
:host(:not([appearance='outlined'i]))
```
as it's the default behaviour, we want to apply its style as a fallback and not by a declarative approach

Reply:
I disagree. All the ghost styles are needed in the outlined option. 
The outlined is based in the ghost but has additional style. 
No ghost properties are overide in the outlined style. It seems right to target the outline for that styles.

Reply:
Rephrase: any ghost-only styles.
Of course common styles stay intact